src/data/tiles.js

```diff
@@ -5,43 +5,31 @@ import { spritesheetImage } from '../helpers/spritesheet'
 
 
 
+function renderStandardSizeTile(context, x, y, isPlacing = 0, canPlace = 1, sourceX, sourceY) {
```

Comment:
does this actually decrease the build size

Reply:
Only by 5b, but yeah, it does.